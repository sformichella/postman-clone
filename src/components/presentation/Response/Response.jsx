import React from "react";
import PropTypes from "prop-types";


export default function Response({ response }) {
  return (
    <p>{response}</p>
  );
}

Response.propTypes = {
  response: PropTypes.string
};
