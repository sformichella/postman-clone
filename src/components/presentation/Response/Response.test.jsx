import React from "react";
import { render, cleanup } from "@testing-library/react";
import Response from "./Response";

describe("Response component", () => {
  afterEach(() => cleanup());
  it("renders Response", () => {
    const { asFragment } = render(<Response />);
    expect(asFragment()).toMatchSnapshot();
  });
});
