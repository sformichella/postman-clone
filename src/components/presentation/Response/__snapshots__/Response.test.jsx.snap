// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Response component renders Response 1`] = `
<DocumentFragment>
  <p />
</DocumentFragment>
`;
